src/core/surface/completion_queue.h

```diff
@@ -51,6 +52,16 @@ typedef struct grpc_cq_completion {
   gpr_uintptr next;
 } grpc_cq_completion;
 
+/** An alarm associated with a completion queue. */
```

Comment:
Not sure... Maybe timer?

On Sun, Oct 4, 2015, 1:51 PM David G. Quintas <notifications@github.com>
wrote:

> In src/core/surface/completion_queue.h
> <https://github.com/grpc/grpc/pull/3618#discussion_r41102234>:
>
> > @@ -51,6 +52,16 @@ typedef struct grpc_cq_completion {
> >    gpr_uintptr next;
> >  } grpc_cq_completion;
> >
> > +/** An alarm associated with a completion queue. */
>
> If this new cq alarms take the grpc_alarm name, what should the current
> grpc_alarm from alarm.h be called?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/3618/files#r41102234>.
>